refs/refs.go

```diff
@@ -41,6 +41,15 @@ func MustParsePlan(s string) Plan {
 	return p
 }
 
+func (p Plan) Less(o Plan) bool {
+	if p.name != o.name {
+		return p.name < o.name
+	}
+	return p.version < o.version
```

Comment:
I would expect this to be numeric sort not lexicographic? E.g. `plan:foo@9` < `plan:foo@10`.

Reply:
It's lexical by design. The version may contain alphanumeric characters like `plan:free@v1`. Tier does not support (by design) semantic versions or numeric-only constraints, so users may have flexibility in how they want to version and sort their plans. The Less function here is convenient for lexical. However, one may use `p.Version()` with sort to sort however they want.